plotly/offline/offline.py

```diff
@@ -236,13 +247,39 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     display(HTML(plot_html))
 
+    if image:
+        warnings.warn('For higher resolution images and more export '
+                      'options, consider making requests to the '
+                      'Plotly image servers. Type help(\'py.image\') '
+                      'for more details.', UserWarning)
```

Comment:
See my flip floppiness in https://github.com/plotly/plotly.py/pull/494/files/f5baa3e6060e9f36676c93797063ea3eb9a4ce91#r66386919

thoughts?